### PR TITLE
add allow ingress from classic param

### PR DESCRIPTION
**Pull request by claymccoy**

### CODE REVIEW

**Comment by claymccoy** [comment]
@cfieber

File: orca-tide/src/main/groovy/com/netflix/spinnaker/orca/tide/tasks/DeepCopyPipelineTask.groovy [hunk 1]
```diff
@@ -36,11 +36,14 @@ class DeepCopyPipelineTask implements Task {
     Map<String, String> source = stage.context.source
     Map<String, String> target = stage.context.target
     def dryRun = stage.context.dryRun ?: false
+    def allowIngressFromClassic = stage.context.allowIngressFromClassic ?: true
```
Comment (cfieber) [inline orca-tide/src/main/groovy/com/netflix/spinnaker/orca/tide/tasks/DeepCopyPipelineTask.groovy]: this is probably not right in either the case of someone trying to pass false, or someone passing the string "true" or "false"

I would make a pojo in there and use stage.mapTo

File: orca-tide/src/main/groovy/com/netflix/spinnaker/orca/tide/tasks/DeepCopyServerGroupTask.groovy [hunk 2]
```diff
@@ -36,7 +36,9 @@ class DeepCopyServerGroupTask implements Task {
     Map<String, String> source = stage.context.source
     Map<String, String> target = stage.context.target
     def dryRun = stage.context.dryRun ?: false
-    def taskId = tideService.deepCopyServerGroup(source.account, source.region, source.asgName, dryRun, target)
+    def allowIngressFromClassic = stage.context.allowIngressFromClassic ?: true
```
Comment (cfieber) [inline orca-tide/src/main/groovy/com/netflix/spinnaker/orca/tide/tasks/DeepCopyServerGroupTask.groovy]: same as above

**Comment by cfieber** [comment]
LGTM

**Pull request merged**